### ISSUE TITLE
[gatsby-source-contentful] Image file extension in capital letters

### ISSUE THREAD

**Issue by xmflsct** [gatsbyjs/gatsby#30069]
## Description

Upgraded to Gatsby `3.0.3` and Contentful plugin `5.0.1`, also using the new `gatsby-plugin-image`.
I get the following warning:
```
Invalid image format "JPG". Supported types are jpg, png and webp"
The resolver for image //images.ctfassets.net/***/***/***/***.JPG returned an invalid value.
```

### Steps to reproduce

See build log here https://github.com/xmflsct/terradelft-website/runs/2048384373

### Expected result

Maybe auto tranform to lower cases?

### Actual result

`.JPG` is not recognised as `.jpg`

### Environment

Run `gatsby info --clipboard` in your project directory and paste the output here.

**Comment by xmflsct** [comment]
https://github.com/gatsbyjs/gatsby/blob/2f599952142b9f2350ef268187839dcbe01ccd74/packages/gatsby-source-contentful/src/extend-node-type.js#L37 This line is where the problem is. Reverting to use `gatsby-image` works fine as before.

**Comment by vladar** [comment]
Published in `gatsby-source-contentful@5.0.2`
